### DISCUSSION TITLE
[Question]: Helm chart Improvements

### DISCUSSION THREAD

**Discussion by pranshu-m** [danny-avila/LibreChat#4010] [Q&A]
### What is your question?

We are using helm-charts to install librechat on our cluster, but I can see a few areas of improvement. But not sure how to proceed on that, is this repo the right place to raise PR or https://github.com/bat-bs/helm-charts/tree/main? 

### More Details

I want add an option to add volumes and volume mounts. In https://github.com/bat-bs/helm-charts/tree/main, one can't disable ingress. We use istio, but disabling ingress gives an error, we also don't want to use incluster MongoDB but disabling that as well gives an error.

### What is the main subject of your question?

Installation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I don't own https://github.com/bat-bs/helm-charts/tree/main and it seems the PR here from that repo is getting stale: https://github.com/danny-avila/LibreChat/pull/3638

If you would like to raise PR to the librechat repo, please feel free to do so.
